recipes/publish.rb

```diff
@@ -29,14 +29,16 @@
 
   # Only build and publish if we have a depot token
   if habitat_depot_token? # ~FC023
-    hab_build node['delivery']['change']['project'] do
-      origin origin
-      plan_dir habitat_plan_dir
-      cwd node['delivery']['workspace']['repo']
-      home_dir delivery_workspace
-      auth_token project_secrets['habitat']['depot_token']
-      depot_url node['habitat-build']['depot-url']
-      action [:build, :publish, :save_application_release]
+    modified_habitat_plan_contexts.each do |plan_context|
```

Comment:
Not sure if it needs to be noted, but this will build everything that's changed in lexical order, which will work fine if the plans in the repo have no dependencies on one another, but will not be what you want if you have dependencies.

In that case you kind of have to do some tricks to implement a poor version of Builder, and reorder `modified_habitat_plan_contexts` to build in the order you want.